client/src/components/screens/Forgot.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { clearAuthErrorMsg } from '../../actions'
+
+import Page from '../Page'
+import RequestResetForm from '../forms/RequestResetForm'
+
+import PageContent from '../styles/PageContentStyles'
+import Headline from '../styles/PageHeadline'
+import ErrorMsg from '../styles/ErrorMsg'
+import InfoMsg from '../styles/InfoMsg'
+
+class Forgot extends Component {
+  componentWillUnmount() {
+    const { auth } = this.props
+
+    if (auth.errorMsg || auth.passwordResetSuccessMsg) {
+      this.props.clearAuthErrorMsg()
```

Comment:
you should use object destucting, like you did for `auth` variable. Do the same for `clearAuthErrorMsg` function:
const { auth, clearAuthErrorMsg } = this.props  , at line 16.

Reply:
Done!